src/steps/video-setup/preview.tsx

```diff
@@ -86,7 +93,15 @@ const PreviewVideo: React.FC<{ input: Input }> = ({ input }) => {
   if (!stream) {
     let inner: JSX.Element;
     if (allowed === false || unexpectedEnd) {
-      inner = <FiAlertTriangle css={{ fontSize: 48, color: COLORS.danger4 }} />;
+      inner = <div>
+        {allowed === false && <ErrorBox
+          css={{ margin: 0 }}
+          title={t(`source-${input.isDesktop ? "display" : "user"}-not-allowed-title`)}
+          body={t(`source-${input.isDesktop ? "display" : "user"}-not-allowed-text`)}
+        />}
+        {/* TODO: differentiate between desktop and camera for better error */}
```

Comment:
What about this ToDo?

Reply:
That was there before and I don't think this PR necessarily needs to fix that.